### DISCUSSION TITLE
How do I submit changes to an open source repo?

### DISCUSSION THREAD

**Discussion by AyyanNiazi** [community/community#22264] [New to GitHub]
<p>hello this is ayyan mern stack intermediate dev<br>
i want to work with open source community please guide<br>
thanks</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/ayyanniazi">@ayyanniazi</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>You can find some suggestions in these references:</p>
<ul>
<li><a href="https://github.community/t/best-practices-for-open-source-software-contributions/10209">Best-practices-for-Open-Source-Software-Contributions</a></li>
<li><a href="https://opensource.guide/" rel="nofollow noopener">opensource.guide</a></li>
<li><a href="https://github.com/Novice-Paradigm/Hello-World" rel="nofollow noopener">Novice-Paradigm/Hello-World</a></li>
<li><a href="https://medium.freecodecamp.org/how-to-start-an-open-source-project-in-new-years-945bad8800d7?gi=50379deeb039" rel="nofollow noopener">how-to-start-an-open-source-project-in-new-years</a></li>
<li><a href="https://up-for-grabs.net/#/" rel="nofollow noopener">up-for-grabs</a></li>
</ul>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>
